README.md

```diff
@@ -512,8 +513,11 @@ API | Description | Auth | HTTPS | CORS |
 | [Lichess](https://lichess.org/api) | Access to all data of users, games, puzzles and etc on Lichess | `OAuth` | Yes | Unknown |
 | [Magic The Gathering](http://magicthegathering.io/) | Magic The Gathering Game Information | No | No | Unknown |
 | [Marvel](http://developer.marvel.com) | Marvel Comics | `apiKey` | No | Unknown |
+| [Minecraft Server Status](https://api.mcsrvstat.us/) | Minecraft Server Status API | Yes | Yes | Unknown |
 | [mod.io](https://docs.mod.io) | Cross Platform Mod API | `apiKey` | Yes | Unknown |
+| [Mojang](https://api.mojang.com) | Mojang API | No | Yes | Unknown |
 | [Open Trivia](https://opentdb.com/api_config.php) | Trivia Questions | No | Yes | Unknown |
+| [Optifine](http://s.optifine.net) | Optifine API, get player statistics | No | No | Unknown |
```

Comment:
This link is not working. Can you remove it please? 😅

Ref: http://s.optifine.net/
![image](https://user-images.githubusercontent.com/50463866/126845968-2f94d03b-1cfa-42ef-9fbf-b2558ca0eb73.png)